### DISCUSSION TITLE
Default placeholder text for grouping drop zone

### DISCUSSION THREAD

**Discussion by spaciandd** [KevinVandy/material-react-table#279] [General]
Currently, there is no property to add the default text for group drop zone, i.e., _"Drag here to group by"_ or I am missing something.

I have seen localisation prop but have not found any default text property for drop zone.

**Comment by KevinVandy** [comment]
It should be in the `localization.dropToGroupBy` option

source file here: https://github.com/KevinVandy/material-react-table/blob/main/packages/material-react-table/src/toolbar/MRT_ToolbarDropZone.tsx

**Reply by spaciandd** [reply]
This text shows while dragging starts. I want to show text even there is no drag start. See in kendo UI,
https://demos.telerik.com/aspnet-mvc/grid/server-grouppaging-virtualization

**Reply by KevinVandy** [reply]
Yeah you're right. Maybe it needs to be handled in state for whether or not drop zone is visible

**Reply by KevinVandy** [reply]
ok, now in v1.5.2, you can accomplish this: https://www.material-react-table.dev/?path=/story/features-column-grouping-examples--column-grouping-drop-zone-always-visible

**Reply by spaciandd** [reply]
@KevinVandy perfect, one more thing I notice is drag handle does not work on my iPhone. If you can fix that as well that will be great.

thank you
